protocol/x/clob/keeper/orders.go

```diff
@@ -208,12 +208,6 @@ func (k Keeper) PlaceShortTermOrder(
 		return 0, 0, err
 	}
 
-	// Validate that adding the order wouldn't exceed subaccount equity tier limits.
-	err = k.ValidateSubaccountEquityTierLimitForShortTermOrder(ctx, order)
-	if err != nil {
```

Comment:
remove the comment on line 170

Reply:
we also technically dont use `ValidateSubaccountEquityTierLimitForShortTermOrder` anymore maybe add a comment that we removed the sa eq tier check for short term orders on check tx to that fn or just del

Reply:
OK I marked it as deprecated.